.circleci/config.yml

```diff
@@ -6,6 +6,19 @@ orbs:
 
 
 jobs:
+  verify:
+    docker:
+      - image: cimg/base:2021.04
```

Comment:
Image maintained by circleci:
https://circleci.com/docs/2.0/circleci-images/

Reply:
Thoughts on retagging?

Reply:
Seems like the main benefit is faster spin up because the image is preloaded by circle. Retagging would remove that benefit

Reply:
My interpretation of that article is that they're on Docker Hub, but by standardizing the base layers and keeping them minimal it makes it more likely they'll be cached already. It doesn't seem to say that Circle is preloading them or something, but that could be the case. Not blocking, it just looked weird to see a non-GS image